### PR TITLE
LA-356  InfluxDB in gates (Kilo backport)

### PR DESCRIPTION
**Pull request by evrardjp**
Backports to Kilo.
See the content in the commits.

Issue: [LA-353](https://rpc-openstack.atlassian.net/browse/LA-353)

Issue: [LA-356](https://rpc-openstack.atlassian.net/browse/LA-356)

### CODE REVIEW

File: scripts/deploy-rpc-playbooks.sh [hunk 1]
```diff
@@ -63,19 +63,17 @@ fi
 # to define its influxdb servers.
 # These playbooks need maas-get to have run previously.
 if [[ "${DEPLOY_TELEGRAF}" == "yes" ]]; then
-    # BUILD_TAG is used as reference for gating, to allow
-    if [[ -z "${BUILD_TAG}" ]]; then
-        # user_zzz_variables are generated at every build, so
+    # BUILD_TAG is used as reference for gating.
+    if [[ -n "${BUILD_TAG}" ]]; then
+        # user_rpco_variables_overrides are generated at every build, so
```
Comment (hughsaunders) [inline scripts/deploy-rpc-playbooks.sh]: user_rpcm_variables

Comment (evrardjp) [inline scripts/deploy-rpc-playbooks.sh]: Done.

**Comment by evrardjp** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by evrardjp** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by evrardjp** [comment]
recheck_trusty_deploy_swift

**Comment by evrardjp** [comment]
recheck_cit_trusty_deploy_swift -- Setuptools issue should be solved inside gating.

**Comment by evrardjp** [comment]
Oh noh, The setuptools issue is NOT in the gates. That won't be solved without a kilo change or an upstream fix.

**Comment by evrardjp** [comment]
Depends on https://github.com/rcbops/rpc-openstack/pull/2411

**Comment by evrardjp** [comment]
Depends on https://github.com/rcbops/rpc-openstack/pull/2414

**Comment by evrardjp** [comment]
recheck_cit_trusty_deploy_swift

**Comment by evrardjp** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by evrardjp** [comment]
Failed post leapfrog on tempest: https://rpc.jenkins.cit.rackspace.net/job/RPC-AIO_kilo-trusty-leapfrogupgrade-swift-pr/36/console

**Comment by evrardjp** [comment]
Twice failed at tempest defcore.

**Comment by mattt416** [comment]
recheck_cit_trusty_leapfrogupgrade_swift - didn't fail on 2nd tempest run this time

**Comment by evrardjp** [comment]
Force merging due to intermittent gating issues.

**Pull request merged**